File: src/main/java/com/jinkyumpark/bookitout/search/apiResponse/seoulLibrary/ApiSeoulLibraryResponse.java
package com.jinkyumpark.bookitout.search.apiResponse.seoulLibrary;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.List;

@NoArgsConstructor @AllArgsConstructor
@Getter
public class ApiSeoulLibraryResponse {
    private String codeMessage;
    private String totalPage;
    private Integer statusCode;

    @JsonProperty("ContentDataList")
    private List<ApiSeoulLibraryBook> bookList;
}

File: search/src/main/kotlin/com/jinkyumpark/search/apiResponse/seoulLibrary/ApiSeoulLibraryResponse.kt
package com.jinkyumpark.search.apiResponse.seoulLibrary

import com.fasterxml.jackson.annotation.JsonProperty

data class ApiSeoulLibraryResponse(
    var codeMessage: String,
    var totalPage: String,
    var statusCode: Int,

    @JsonProperty("ContentDataList")
    var bookList: List<ApiSeoulLibraryBook>,
)
